css/src/atomics/typography.scss

```diff
@@ -161,3 +161,17 @@
 .text-transform-uppercase {
 	text-transform: uppercase !important;
 }
+
+// Alignment
+
+.text-align-left {
```

Comment:
Default, needed?

Reply:
There are references in docs that need this class.

Reply:
Mind explaining the use case?

Reply:
Discussed offline, removed left alignment class, including on tablet.